css-two-column-layout/styles.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Karla', sans-serif;
+}
+
+.container {
+  align-items: center;
+  margin: auto;
+  justify-content: center;
+  width: 900px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.column-full {
+  width: 100%;
+  padding-left: 4rem;
+  padding-right: 4rem;
+}
+
+.column-half {
+  width: 49%;
+  display: flex;
```

Comment:
If you are applying `display: flex` to a column class, that usually means that the `column` should contain a `row` class element. That is the purpose of the `row` class, to make flex containers